packages/amplify-graphql-auth-transformer/src/__tests__/auth-operations.test.ts

```diff
@@ -0,0 +1,284 @@
+import { ModelTransformer } from '@aws-amplify/graphql-model-transformer';
+import { ConflictHandlerType, GraphQLTransform, SyncConfig } from '@aws-amplify/graphql-transformer-core';
+import { AppSyncAuthConfiguration } from '@aws-amplify/graphql-transformer-interfaces';
+import { SearchableModelTransformer } from '@aws-amplify/graphql-searchable-transformer';
+import { AccessControlMatrix } from '../accesscontrol';
+import { AuthTransformer } from '../graphql-auth-transformer';
+import { MODEL_OPERATIONS } from '../utils';
+
+test('invalid granular read operation at the field level', () => {
+  const authConfig: AppSyncAuthConfiguration = {
+    defaultAuthentication: {
+      authenticationType: 'AMAZON_COGNITO_USER_POOLS',
+    },
+    additionalAuthenticationProviders: [{
+      authenticationType: 'API_KEY',
+    }],
+  };
+  const invalidSchema = `
+    type Test @model @auth(rules: [{ allow: public, operations: [ list, create ]}]) {
+      id: ID!
+      name: String @auth(rules: [{ allow: public, operations: [ get, create ]}])
+    }`;
+  const transformer = new GraphQLTransform({
+    authConfig,
+    transformers: [new ModelTransformer(), new AuthTransformer()],
+  });
+  expect(() => transformer.transform(invalidSchema)).toThrowError(
+    '\'get\' operation is not allowed at the field level.',
```

Comment:
I thought list wasn't allowed at the field level, you can still allow a 'get', right? or are you required to use 'read' here?

Reply:
at the field level we decided to only allow read (none of the more granular level access operations)